.chloggen/oracledb-max-tablespace-size-unset.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: oracledbreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: return -1 if max tablespace size is not set
```

```suggestion
note: record a value of -1 for `oracledb.tablespace_size.limit` if the max tablespace size is not set
```
I think it would help to clarify which metric is being changed here.